1-mandatory/2-people-i-know.js

```diff
@@ -403,6 +440,16 @@ This time, I only want the full names of the people who can multitask
 
 let colleaguesWhoCanMultitask = [];
 
+function getFriendsWhoMultitask(friend) {
+  for (let colleague of friend.colleagues) {
+    if (colleague.skills.includes("Multi-tasking")) {
+      colleaguesWhoCanMultitask.push(colleague.name);
+    }
+  }
+}
+
+friends.forEach(getFriendsWhoMultitask);
+
 /* ======= TESTS - DO NOT MODIFY ===== 
```

Comment:
i can't see here that last action (friends.forEach) gives the value to coleaguesWhoCanMultitask array